doc/source/drivers/vector/xodr.rst

```diff
@@ -0,0 +1,198 @@
+* *RoadSignal*: Polygonal surface (TIN) of the road signal mesh as :cpp:class:`OGRTriangulatedSurface`.
+
+Spatial reference
+-----------------
+
+By definition, OpenDRIVE geometries are always referenced in a Cartesian coordinate system which defaults to having its origin at ``(0, 0, 0)``. If real-world coordinates are used, the OpenDRIVE header provides a PROJ.4 string with the definition of a projected spatial reference system:
+
+::
+
+  <header ...>
+    <geoReference><![CDATA[+proj=tmerc +lat_0=0 +lon_0=9 +k=0.9996 +x_0=500000 +y_0=0 +datum=WGS84 +units=m +no_defs]]></geoReference>
+  </header>
+
+The XODR driver uses this PROJ definition as spatial reference for creation of all OGR geometry layers. 
+
+Limitations
+-----------
+
+By default, OpenDRIVE XML files are opened by GDAL always in UTF-8 encoding, see :cpp:func:`VSIFOpenExL`.
+
+Open options
+------------
+
+The following open options can be specified
+(typically with the ``-oo name=value`` parameters of :program:`ogrinfo` or :program:`ogr2ogr`):
+
+-  .. oo:: EPS
+      :choices: <float>
+      :default: 1.0
+
+      Epsilon value for linear approximation of continuous OpenDRIVE geometries. A smaller value results in a finer sampling. This parameter is internally forwarded to libOpenDRIVE.
+
+-  .. oo:: DISSOLVE_TIN
+      :choices: YES, NO
+      :default: NO
+
+      Whether to dissolve triangulated surfaces. By setting this option to YES, the TIN layers *Lane* and *RoadMark* of geometry type :cpp:class:`OGRTriangulatedSurface` will be simplified to single, simple :cpp:class:`OGRPolygon` geometries. This performs a :cpp:func:`UnaryUnion` which dissolves boundaries of all touching triangle patches and thus yields a slimmer dataset which often suffices for basic GIS usage. Be aware that this dissolving step increases processing time significantly.
+      Layer *RoadSignal* will be dissolved to a simple :cpp:class:`OGRPoint`.
+
+Examples
+--------
+
+- Translate OpenDRIVE road *ReferenceLine* elements (``<planView>``) to :ref:`Shapefile <vector.shapefile>` using :program:`ogr2ogr`. The desired :ref:`layer type <layer types>` which is to be extracted from the dataset is specified as the last parameter of the function call. 
+
+  ::
+
+    ogr2ogr -f "ESRI Shapefile" CulDeSac.shp CulDeSac.xodr ReferenceLine
+
+- Convert the whole OpenDRIVE dataset with all its different layers into a :ref:`GeoPackage <vector.gpkg>` using:
+
+  ::
+
+    ogr2ogr -f "GPKG" CulDeSac.gpkg CulDeSac.xodr
+
+- Convert the whole OpenDRIVE dataset with custom parameters :oo:`EPS` and :oo:`DISSOLVE_TIN` into a :ref:`GeoPackage <vector.gpkg>`:
+
+  ::
+
+    ogr2ogr -f "GPKG" CulDeSac.gpkg CulDeSac.xodr -oo EPS=0.9 -oo DISSOLVE_TIN=YES
+
+Convenient usage through docker image 
+-------------------------------------
+
+To use the XODR driver inside a docker image, first build the image from the corresponding docker image directory 
```

Comment:
Please remove the docker/ubuntu-small/DockerfileXODR file, and move the building of OpenDrive into the docker/ubuntu-full (full) build recipee. It would also be fine to move it as well in the alpine-normal (assuming OpenDrive is not too big). But we definitely don't want a new Dockerfile to maintain. 4 is already more than enough :-)